snuba/datasets/entities/sessions.py

```diff
@@ -310,7 +265,8 @@ def __init__(self) -> None:
             storages=[storage],
             query_pipeline_builder=SimplePipelineBuilder(
                 query_plan_builder=StorageQueryPlanBuilder(
-                    storages=[StorageAndMappers(storage, TranslationMappers())]
+                    storages=[StorageAndMappers(storage, TranslationMappers())],
+                    selector=DefaultQueryStorageSelector(),
```

Comment:
How is it possible this is using the `DefaultQueryStorageSelector`? Shouldn't it be using the sessions selector?

Reply:
`ORG_SESSIONS` only uses the org materialized sessions storage so the default is possible. `SESSIONS` on the other hand requires the storage selector.

Reply:
Ah got it. OK cool.